06-service-recommender/backend/src/common/components.py

```diff
@@ -0,0 +1,23 @@
+"Components cannot be defined in pipelines themselves, so define them here."
```

Comment:
This got me stuck for a while because I defined the class in the `pipeline_wrapper.py` file itself.